pkg/telegram-bot/telegram-bot.go

```diff
@@ -1 +1,421 @@
 package telegrambot
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/joho/godotenv"
+	"gopkg.in/telebot.v4"
+)
+
+const (
+	USER       = "User"
```

Comment:
if you donot want to use it out side of this file, you can use lower case like:
var (
        user = "user"
)